.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Bug
+about: Bug report
+title: ''
+labels: status/triage, type/bug
+assignees: ''
+
+---
+### Summary
+<!--- Please provide a general summary of the issue. -->
+
+
+---
+
+### Reproduction
+
+##### Steps
+<!--- What steps should be taken to reproduce the issue? -->
+
+1.
+2.
+3.
+
+##### Expected
+<!--- What did you expect to happen -->
+
+
+---
+
+### Environment
+
+##### pack report
```

Comment:
Lifecycle bugs might show up in platforms other than `pack`. Instead of `pack report` and `docker info`  we could put something more generic like `platform` and then provide `pack` and `pack report` as an example of how this could be filled out.